lib/hammer_cli_katello/repository.rb

```diff
@@ -478,10 +480,14 @@ def import_uploads(uploads, opts = {})
       end
 
       def print_results(name, results)
-        if results.empty? || results.dig('output', 'upload_results').empty?
+        if results.empty? || results.dig('output', 'upload_results')&.empty?
           print_message _("Successfully uploaded file '%{name}'") % {
             :name => name
           }
+        elsif results["pending"]
+          print_message _("Started uploading file '%{name}'") % {
+            :name => name
```

Comment:
I think we should poll the task to completion like we do with other tasks:    https://github.com/Katello/hammer-cli-katello/blob/master/lib/hammer_cli_katello/content_view.rb#L190

They use that HammerCLIForemanTasks::Async module to do all the polling for us.  (I'm not sure how hard that is to use here)